### PR TITLE
Theodw 1380 nsip project entity excluding cases

### PR DESCRIPTION
**Pull request by IsminiA**
- changed the variable that creates casereference from horizon. new variable is caseNodeId . This has been implemented in workspace/notebook/py_sb_horizon_harmonised_nsip_project.json

### CODE REVIEW

File: workspace/notebook/py_sb_horizon_harmonised_nsip_project.json [hunk 1]
```diff
@@ -312,7 +312,7 @@
 					"                        ,applicantwebaddress\n",
 					"                        ,applicantphonenumber\n",
 					"                        ,applicantdescriptionofproject\n",
-					"                        ,caseNodeId\n",
+					"                       --- ,caseNodeId\n",
```
Comment (muhammadkc) [inline workspace/notebook/py_sb_horizon_harmonised_nsip_project.json:315]: We are losing a column, HorizonCaseNumber, I think it needs to be added to the query

Comment (IsminiA) [reply workspace/notebook/py_sb_horizon_harmonised_nsip_project.json]: updated and included HorizonCaseNumber

File: workspace/notebook/py_sb_horizon_harmonised_nsip_project.json [hunk 2]
```diff
@@ -781,7 +783,7 @@
 					"                    ,IFNULL(CAST(applicantwebaddress AS String), '.')\n",
 					"                    ,IFNULL(CAST(applicantphonenumber AS String), '.')\n",
 					"                    ,IFNULL(CAST(applicantdescriptionofproject AS String), '.')\n",
-					"                    ,IFNULL(CAST(caseNodeId AS String), '.')\n",
+					"                   --,IFNULL(CAST(HorizonCaseNumber AS int), '.')\n",
```
Comment (muhammadkc) [inline workspace/notebook/py_sb_horizon_harmonised_nsip_project.json:786]: Should this be commented? just curious

Comment (IsminiA) [reply workspace/notebook/py_sb_horizon_harmonised_nsip_project.json:786]: if you mean line 786, no it shouldnt be commented

Comment (IsminiA) [reply workspace/notebook/py_sb_horizon_harmonised_nsip_project.json]: done now :)

**Review by muhammadkc — APPROVED** [review]

**Review by ChrisToppingKandC — APPROVED** [review]

**Pull request merged**